types/src/transaction/transaction_v1.rs

```diff
@@ -1,20 +1,32 @@
+pub mod arg_handling;
 mod errors_v1;
-mod transaction_v1_body;
+pub mod fields_container;
 #[cfg(any(feature = "std", test))]
 mod transaction_v1_builder;
-mod transaction_v1_category;
 mod transaction_v1_hash;
-mod transaction_v1_header;
+pub mod transaction_v1_payload;
 
-#[cfg(any(all(feature = "std", feature = "testing"), test))]
-use alloc::string::ToString;
-use alloc::{collections::BTreeSet, vec::Vec};
-use core::{
-    cmp,
-    fmt::{self, Debug, Display, Formatter},
-    hash,
+use crate::{
+    bytesrepr::{self, Error, FromBytes, ToBytes},
+    crypto,
 };
+#[cfg(any(feature = "testing", test))]
```

Comment:
```suggestion
#[cfg(any(all(feature = "std", feature = "testing"), test))]
```
I believe same as random_auction

Reply:
I really don't get it why we add "std" to code that is "std" agnostic, but OK I'll make it the same for the sake of consistency.

Reply:
it's just cargo lint and fmt that complains because random_auction / random_install_upgrade is flag std